alkos/kernel/src/test_framework.cpp

```diff
@@ -0,0 +1,61 @@
+/* internal includes */
+#include <arch_utils.hpp>
+#include <terminal.hpp>
+#include <test_framework.hpp>
+
+TODO_BY_THE_END_OF_MILESTONE0
```

Comment:
To be honest I have no idea what milestone0 is.
We haven't really defined milestones in our project.

Reply:
I forgot to mention that during sync call. We should talk about that for sure